qa/tasks/ses_qa.py

```diff
@@ -146,13 +146,16 @@ def drive_replace_initiate(self, **kwargs):
         In order to "hide" an existing disk from the ceph.c_v in teuthology
         the disk is formatted and mounted.
         """
-        # :TODO: make this random and find correct disk device
-        osd_id = "2"
+        from random import randint
```

Comment:
I don't understand why we need be random here?

Reply:
to avoid hardcoding drives. Also we've uncovered a bug : https://github.com/SUSE/DeepSea/pull/1722

Reply:
no, question is why is it randomized, randomization is not always good, in this case you can try and take last drive or the first?